crates/schema/src/identifier.rs

```diff
@@ -125,6 +130,10 @@ mod tests {
     proptest! {
         #[test]
         fn test_standard_ascii_identifiers(s in "[a-zA-Z_][a-zA-Z0-9_]*") {
+            // Ha! Proptest will reliably find these.
+            if Identifier::is_reserved(&s) {
+                return Ok(());
+            }
```

Comment:
Proptest has a way of skipping cases: https://docs.rs/proptest/latest/proptest/test_runner/enum.TestCaseError.html .

Reply:
https://docs.rs/proptest/latest/proptest/macro.prop_assume.html